lib/Drush/Boot/DrupalBoot8.php

```diff
@@ -118,15 +120,26 @@ function bootstrap_drupal_database() {
   }
 
   function bootstrap_drupal_configuration() {
+    global $drupal_update;
     $this->request = Request::createFromGlobals();
     $classloader = drush_drupal_load_autoloader(DRUPAL_ROOT);
     // @todo - use Request::create() and then no need to set PHP superglobals
     $kernelClass = new \ReflectionClass('\Drupal\Core\DrupalKernel');
     if ($kernelClass->hasMethod('addServiceModifier')) {
-      $this->kernel = DrupalKernel::createFromRequest($this->request, $classloader, 'prod', DRUPAL_ROOT);
+      if (!$drupal_update) {
```

Comment:
Rather than having Drush bootstrap and then call commands, I think it would be better to have commands call `$bootstrap->...` as needed. Maybe for Drush 10; keeping this parameterized is fine for now.